theme/blockly-core.less

```diff
@@ -396,40 +397,72 @@ text.blocklyCheckbox {
         Focus styles
 *******************************/
 
-// Avoid focus outlines when not using the keyboard navigation plugin.
-div.blocklyTreeRoot > div > div[role="tree"]:focus-visible {
-    outline: none;
+
+html {
```

Comment:
CSS as per plugin test page. Probably not its ultimate home but a copy is needed here for now.